_RFCs/002-node-scaffolding-split.md

```diff
@@ -0,0 +1,60 @@
+- Change Name: node-scaffolding-split
+- Start Date: 2017-11-20
+- RFC PR:
+- Core-Plans Issue:
+
+# Summary
+[summary]: #summary
+
+This RFC proposes splitting the current [scaffolding-node](https://github.com/habitat-sh/core-plans/tree/master/scaffolding-node) into two scaffoldings - one focused on applications that run dynamically and one focused on applications that are compiled and run as static assets.
+
+# Motivation
+[motivation]: #motivation
+
+There are currently two ways of running Node applications. Some applications (especially those created with the [Express](https://expressjs.com/) framework) run the application using Node.js's web server. Others - especially those built with [React](https://reactjs.org/) and [Angular](https://angular.io/) - compile Javascript, etc. as static assets and run them behind a proxy like Nginx. Supporting both of these approaches through one scaffolding would be deeply complex at best and untenable at worst.
```

Comment:
It's worth pointing out that this will be true of Ruby-based static-site frameworks as well such as Middleman, Jekyll and the like.

Reply:
True!  Focusing on Node for this RFC, but will keep that in mind :)